plugins/ua/universalarchive.plugin.zsh

```diff
@@ -0,0 +1,59 @@
+alias ua='universalarchive'
```

Comment:
This won't get loaded, the appropriate plugin path nomenclature is `plugins/<name>/<name>.plugin.zsh`. You should reconsider whether you want it to be named `ua` or `universalarchive`. If you want the latter, it should be `plugins/universalarchive/universalarchive.plugin.zsh`; otherwise it should be `plugins/ua/ua.plugin.zsh`.